clouddriver-dcos/src/test/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/validators/servergroup/DeployDcosServerGroupDescriptionValidatorSpec.groovy

```diff
@@ -43,12 +44,13 @@ class DeployDcosServerGroupDescriptionValidatorSpec extends Specification {
 
   void "validate should give errors when given an invalid DeployDcosServerGroupDescription"() {
     setup:
-      def description = new DeployDcosServerGroupDescription(credentials: new DcosCredentials(null, null, null, null, null, null),
+      def description = new DeployDcosServerGroupDescription(region: "default", credentials: new DcosCredentials(null, null, null, null, null, null),
```

Comment:
You moved this literal into a variable for one test but not the others, probably should move into a variable for all.